app/src/main/java/edu/usf/sas/pal/muser/util/MusicServiceUtils.java

```diff
@@ -0,0 +1,24 @@
+package edu.usf.sas.pal.muser.util;
+
+import android.support.annotation.Nullable;
+
+import com.simplecity.amp_library.model.Song;
+import com.simplecity.amp_library.utils.MusicServiceConnectionUtils;
+
+public class MusicServiceUtils {
```

Comment:
Is there any reason not to move the below two methods to `MusicServiceConnectionUtils` instead of this new class?

If we need a new class, I'd prefer it to be in Kotlin rather than Java.

Reply:
I was thinking if we have our utils class of our own in our package `edu.usf.sas.pal.muser`, it'd be easier for us to differentiate between our code and the legacy "Shuttle" code in the long run. And yes, I can covert this to kotlin in the future commit. But, feel free to let me know if you would like to me move these to the original `MusicServiceConnectionUtils`.

Reply:
Let's go ahead and move to `MusicServiceConnectionUtils`. It seems like these methods should belong there anyway.